presto-hive/src/test/java/com/facebook/presto/hive/TestShowStats.java

```diff
@@ -33,15 +33,10 @@ public TestShowStats()
     public void setUp()
     {
         assertUpdate("CREATE TABLE nation_partitioned(nationkey BIGINT, name VARCHAR, comment VARCHAR, regionkey BIGINT) WITH (partitioned_by = ARRAY['regionkey'])");
+        assertUpdate("CREATE TABLE orders(orderkey BIGINT, totalprice BIGINT, categoryid BIGINT) WITH (partitioned_by = ARRAY['categoryid'])");
```

Comment:
Why do we need this table? Can we use non partitioned nation?

Reply:
@picrinite Vic, what's the answer here? Why do you need `orders` table?